src/main/java/org/trips/service_framework/controllers/BaseCmsController.java

```diff
@@ -0,0 +1,49 @@
+package org.trips.service_framework.controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.http.HttpStatus;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.trips.service_framework.aop.Authenticate;
+import org.trips.service_framework.dtos.CmsSearchRequestBody;
+import org.trips.service_framework.dtos.CmsSkuResponse.Sku;
+import org.trips.service_framework.dtos.SkuAttributes;
+import org.trips.service_framework.helpers.CmsHelper;
+import org.trips.service_framework.models.responses.StatusResponse;
+import org.trips.service_framework.responses.CmsResponse;
+import org.trips.service_framework.services.CmsService;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/cms")
+@RequiredArgsConstructor
+public class BaseCmsController {
```

Comment:
Rename it to CmsController. Base terminology is used in case if this class is getting inherited which is not the case here.